src/main/java/mate/academy/bookstore/model/order/OrderItem.java

```diff
@@ -0,0 +1,45 @@
+package mate.academy.bookstore.model.order;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+import mate.academy.bookstore.model.Book;
+import org.hibernate.annotations.SoftDelete;
+
+@Getter
+@Setter
+@ToString
+@EqualsAndHashCode(of = {"id", "order", "book", "price"})
+@SoftDelete
+@Entity
+@Table(name = "order_items")
+public class OrderItem {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "order_id", nullable = false)
+    private Order order;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "book_id", nullable = false)
+    private Book book;
+
+    @Column(nullable = false)
+    private int quantity;
```

Comment:
```suggestion
    private Integer quantity;
```
I prefer to use the wrapped type as you can tell if the object has been saved/loaded to/from the database whether or not the id value is null.